ceph-pull-requests-arm64/config/definitions/ceph-pull-requests-arm64.yml

```diff
@@ -7,7 +7,7 @@
           - ../../../scripts/build_utils.sh
           - ../../build/build
     concurrent: true
-    disabled: false
+    disabled: true
```

Comment:
can we instead add
```yaml
white-list-labels:
            - arm
```

Reply:
what is the difference?

my goal is to make the little dot next a PR green (instead of red every time, requiring you to open the PR page to see whether it was the arm test or something important that failed)

Reply:
the difference is that this job is disabled by default. and it's triggered only if "arm64" label is added.

Reply:
:+1:

Reply:
Fixed